resources/metadata/schemaPatch.yaml

```diff
@@ -0,0 +1,26 @@
+metadata:
+  name: schemaPatch
```

Comment:
meta: this is not limited to json schema, this is a general "apply this patch to any json" step. Maybe we should name it "jsonPatch" and hint to the use-case of schema patching in the docs?